### PR TITLE
tlky conference interview dhammadina

### PR DESCRIPTION
**Pull request by JJM317**
Stuggled with this one. Not sure if my choices were are correct. This is a blog of an interview. Let me know what changes you think are necessary. Thanks
~Joe

### CODE REVIEW

File: _content/reference/tlky-international-conference-2021-interview-dhammadina_raymond-lam.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "The TLKY International Conference 2021 Interview Series – Ven. Bhikkhuni Dhammadinna"
+authors:
+  - "Raymond Lam"
```
Comment (buddhist-uni) [inline _content/reference/tlky-international-conference-2021-interview-dhammadina_raymond-lam.md:4]: Put dhammadinna as first author (Lam as second or as editor)

File: _content/reference/tlky-international-conference-2021-interview-dhammadina_raymond-lam.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+external_url: "https://teahouse.buddhistdoor.net/the-tlky-international-conference-2021-interview-series-ven-bhikkhuni-dhammadinna/"
+drive_links:
+  - "https://drive.google.com/file/d/19I-xjtDOvpO2DkBCcGC8DObyBDvECs8b/view?usp=sharing"
+course: epistemology
+tags:
+  - phenomenology
+  - nibbāna
```
Comment (buddhist-uni) [inline _content/reference/tlky-international-conference-2021-interview-dhammadina_raymond-lam.md:11]: no accents in tag slugs

File: _content/reference/tlky-international-conference-2021-interview-dhammadina_raymond-lam.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: "The TLKY International Conference 2021 Interview Series – Ven. Bhikkhuni Dhammadinna"
+authors:
+  - "Raymond Lam"
+external_url: "https://teahouse.buddhistdoor.net/the-tlky-international-conference-2021-interview-series-ven-bhikkhuni-dhammadinna/"
+drive_links:
+  - "https://drive.google.com/file/d/19I-xjtDOvpO2DkBCcGC8DObyBDvECs8b/view?usp=sharing"
+course: epistemology
+tags:
```
Comment (buddhist-uni) [inline _content/reference/tlky-international-conference-2021-interview-dhammadina_raymond-lam.md:9]: `modern` also

**Review by buddhist-uni — COMMENTED** [review]
This should go in essays or articles.

This interview discusses the paper that you added before right? That should be cross linked if so.

**Comment by JJM317** [comment]
> right

**Comment by JJM317** [comment]
I tried making the commits. Not sure if they went through since I saw that
the PR was closed after I submitted. I also put in the link to the other
paper with a message to you, if that went through.

I saw the Pramana section on the website. Looks great.

~Joe

**Comment by buddhist-uni** [comment]
Hi Joe, it seems you closed the PR somehow and no the changes haven't gone through. I'll go ahead and reopen this, see if you can push your commit?

Also: yes. I normally wait for everything to be added before launching but I really wanted to get this out in time for the end of January email (which couldn't wait due to TinyLetter shutting down next month).

**Comment by JJM317** [comment]
I opened the reopened PR and the changes I made are there, i.e changed to
essays, dhammadina added as an author, added modern tag, and I linked to
the previous Dhammadina article. I made a small change (deleted a space) so
I could submit the commit (screenshot below).

I had a question about the linked article. It seems that the conference
paper is the published article, but I don't see anything saying as much in
the article. I put a line in the description and linked to the article,
though I can't 100% confirm.

[image: image.png]

**Pull request merged**